poppy/utils.py

```diff
@@ -330,8 +332,18 @@ def display_EE(HDUlist_or_filename=None,ext=0, overplot=False, ax=None, mark_lev
         HDUlist = HDUlist_or_filename
     else: raise ValueError("input must be a filename or HDUlist")
 
+    im = HDUlist[ext].data.copy() # don't change normalization of actual input array, work with a copy!
 
-    radius, profile, EE = radial_profile(HDUlist, EE=True)
+    if normalize.lower() == 'peak':
+        _log.debug("Displaying PSF normalized to peak = 1")
+        im /= im.max()
+    elif normalize.lower() =='total':
+        _log.debug("Displaying PSF normalized to PSF total = 1")
+        im /= im.sum()
+        
+    HDUlist_copy = fits.HDUList([fits.PrimaryHDU(data=im,header=HDUlist[ext].header)])
```

Comment:
This could be simplified using `deepcopy` from the standard library: 

```
import copy
HDUlist_copy = copy.deepcopy(HDUlist)
im = HDUlist[ext].data
```